lua/astrocommunity/pack/sql/init.lua

```diff
@@ -45,7 +86,11 @@ return {
               desc = "Load sqls.nvim with sqls",
               callback = function(args)
                 local client = assert(vim.lsp.get_client_by_id(args.data.client_id))
-                if client.name == "sqls" then require("sqls").on_attach(client, args.buf) end
+                if client.name == "sqls" then
+                  -- Disable formatting due to bugs: https://github.com/sqls-server/sqls/issues/149
+                  client.server_capabilities.documentFormattingProvider = false
```

Comment:
should this be done here or should this be done in AstroLSP?

Reply:
I couldn't figure out how to do it in LSP 😄

Reply:
I resolved this